frontends/web/src/locales/en/app.json

```diff
@@ -537,7 +537,7 @@
     },
     "expert": {
       "factoryReset": {
-        "description": "Reset your device to factory settings. This deletes the wallet from your BitBox02!",
+        "description": "Reset your device to factory defaults - this will erase your BitBox02 wallet!",
```

Comment:
Nit:

I am not sure what this change was about. Before it was more clear IMO that the wallet is only erased from the device. Now it could be misinterpreted that the wallet will be gone forever.
Also it doesn't really reset to factory defaults. Factory default has no firmware.

Reply:
I agree, but I think there has already been some bike-shedding on this :sweat_smile:

Reply:
I reverted it